include/bluetooth/audio.h

```diff
@@ -968,6 +971,23 @@ int bt_audio_chan_unlink(struct bt_audio_chan *chan1,
  */
 int bt_audio_chan_send(struct bt_audio_chan *chan, struct net_buf *buf);
 
+/** @brief Create audio unicast group.
+ *
+ *  Create a new audio unicast group with one or more audio channels as a
+ *  cental. Channels in a unicast group shall share the same interval, framing
+ *  and latency (see @ref bt_codec_qos).
+ *
+ *  @param[in]  chans          Array of channel objects being used for the
+ *                             group.
+ *  @param[in]  num_chan       Number of channels in @p chans.
+ *  @param[out] unicast_group  Pointer to the unicast group created
+ *
+ *  @return Zero on success or (negative) error code otherwise.
+ */
+int bt_audio_unicast_group_create(struct bt_audio_chan *chans,
+				  uint8_t num_chan,
```

Comment:
Alignment seems off - but that may just be Github

Reply:
Looks correct in my editor at least :D